### PR TITLE
Add tutorial article on how to customize a form for a SharePoint list

### PR DESCRIPTION
**Pull request by Steven-Jia**
## Category

- [x] Content fix
- [x] New article

## What's in this Pull Request?

> New tutorial on how to customize a form for a SharePoint list

### CODE REVIEW

**Comment by Steven-Jia** [comment]
@jackwi-alt @chakkaradeep for review

File: docs/business-apps/power-apps/get-started/create-your-first-custom-form.md [hunk 1]
```diff
@@ -37,105 +40,110 @@ Before proceeding with these instructions, make sure your SharePoint site is set
...
-2. To customize the look and feel of the image, you can set its properties.
-    > When a control on the canvas is selected, the Properties pane that is associated with the control opens on the right-hand panel.
+2. To customize the appearance of the image, you can set its properties.
+
+    1. When you select a control on the canvas, on the right-hand panel, the **Properties** pane associated with the control appears.
 
     ![Image control with Properties panel](../../../images/gs01-image-control-selected.png)
 
-    Since the image will be the profile photo of the owner, let us make it the same height as the **Owner_DataCard1** [data card](https://docs.microsoft.com/en-us/powerapps/maker/canvas-apps/working-with-cards).
+    2. Because the image is the profile photo of you (the owner), we recommend you make it the same height as the **Owner_DataCard1** [data card](https://docs.microsoft.com/en-us/powerapps/maker/canvas-apps/working-with-cards).
```
Comment (Steven-Jia) [inline docs/business-apps/power-apps/get-started/create-your-first-custom-form.md:51]: The image if the profile photo of the selected item, so it probably is not the reader of this article

**Comment by chakkaradeep** [comment]
@Steven-Jia @jackwi-alt reason we are not able to merge is there are conflicts. Can anyone of you resolve the conflicts? I think there could be some race condition when the file was committed.

**Comment by Steven-Jia** [comment]
@VesaJuvonen @andrewconnell please let me know if there are any other processes that I need to follow to get this PR merged. Thanks!

**Comment by andrewconnell** [comment]
Confused... I was under the impression @chakkaradeep owned this content and we were hands-off. Is that not correct?

I can review to ensure it meets MSDocs req's... is it ready for that? If so I'll do that now. But new content must be merged by MSFT, not me.

File: docs/business-apps/power-apps/get-started/create-your-first-custom-form.md [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+
+## Required setup
+
+Before starting, make sure your SharePoint site is set up with the [required lists and libraries](https://docs.microsoft.com/sharepoint/dev/business-apps/get-started/set-up-sharepoint-site-lists-libraries).
+
+## Create a custom form
+
+1. In your SharePoint site, browse to the **Projects** list.
+
+2. To create and manage flows for a list or a library, from the command bar, on the list or library page, select **Power Apps**. More options appear:
```
Comment (andrewconnell) [inline docs/business-apps/power-apps/get-started/create-your-first-custom-form.md:23]: Don't use numbers in bullets... all numbered bullets should start with `1.`. Let the rendering engine add the actual numbers. This simplifies maintenance going forward.

File: docs/business-apps/power-apps/get-started/create-your-first-custom-form.md [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+
+    ![Create a custom form from Projects list](../../../images/gs01-create-a-custom-form-command-bar.png)
+
+3. Select **Customize forms**. The Power Apps studio appears and loads your form onto the canvas. If the **Welcome to Power Apps Studio** dialog box opens, select **Skip**.
+
+    ![Power Apps studio with custom form](../../../images/gs01-power-apps-studio-custom-form.png)
+
+### Add a user profile photo
+
+1. On the **Insert** tab, select **Media** > **Image**. An [Image](https://docs.microsoft.com/en-us/powerapps/maker/canvas-apps/controls/control-image) control named **Image1** is added to the canvas and to the [Tree view](https://docs.microsoft.com/en-us/powerapps/maker/model-driven-apps/using-tree-view-on-form) on the left-hand panel.
```
Comment (andrewconnell) [inline docs/business-apps/power-apps/get-started/create-your-first-custom-form.md:37]: Don't include locales in URLs... the site uses the user's browser agent to redirect them to localized pages. This is present in multiple places... I'm only going to add the comment here but applies to entire PR.

File: docs/business-apps/power-apps/get-started/create-your-first-custom-form.md [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+
+    ![Add an image in Power Apps studio](../../../images/gs01-power-apps-studio-add-image.png)
+
+2. To customize the appearance of the image, you can set its properties.
+    
+    a. When you select a control on the canvas, on the right-hand panel, the **Properties** pane associated with the control appears.
```
Comment (andrewconnell) [inline docs/business-apps/power-apps/get-started/create-your-first-custom-form.md:43]: Same comment above with numbered bullets applies to sub-bullets. If you'd use `<ol>` in HTML, use `1.` in markdown.

File: docs/business-apps/power-apps/get-started/create-your-first-custom-form.md [hunk 5]
```diff
@@ -0,0 +1,141 @@
...
+
+    Alternatively, in the Tree view, right-click **Attachments_DataCard1**, and select **Unlock**.
+
+2. Toggle back to the **Properties** pane, and select the **Visible** field. The formula bar updates to display this property.
+
+    ![Select visible field](../../../images/gs01-attachments-property-visible.png)
+
+3. In the formula bar, enter the following formula:
+
+   If(SharePointIntegration.Selected.Owner.DisplayName = "Nestor Wilke", false)
```
Comment (andrewconnell) [inline docs/business-apps/power-apps/get-started/create-your-first-custom-form.md:111]: Should this be formatted as inline code? It appears to be code...

File: docs/business-apps/power-apps/get-started/create-your-first-custom-form.md (hunk 2 above)
Comment (Steven-Jia) [reply docs/business-apps/power-apps/get-started/create-your-first-custom-form.md:23]: I'm confused. What is the requested change here?

Comment (Steven-Jia) [reply docs/business-apps/power-apps/get-started/create-your-first-custom-form.md:23]: The bullets below (create an app, etc.) aren't using numbers... and the instruction steps are using numbers. Does one or both of them need to be changed?

Comment (andrewconnell) [reply docs/business-apps/power-apps/get-started/create-your-first-custom-form.md:23]: @steven-jia lines 21 & 23 in the file are markdown bullets... one is "1." and the other is "2.". When doing markdown bullets, don't use explicit numbers... every single bullet should be "1." and you should rely on the MSDocs markdown rendering engine to render the correct numbers. This simplifies maintenance going forward.

You can preview this in VSCode... if the numbers aren't rendering the correct sequence or are restarting, there's an issue with your indentation that should be resolved.

Look at this [file for a good example](https://github.com/SharePoint/sp-dev-docs/blob/master/docs/spfx/web-parts/get-started/connect-to-sharepoint.md). When you see the rendered bullets, they are a sequence of numbers (1-2-3-4...). However, if you look at the raw source, all you see is (1. 1. 1. 1.).

Comment (Steven-Jia) [reply docs/business-apps/power-apps/get-started/create-your-first-custom-form.md]: Thanks for the clarification! I've made the changes to numbering.

**Comment by VesaJuvonen** [comment]
I'll run this through the validations one more time and get it included in the TOC with possible edits. Thanks @andrewconnell and @Steven-Jia for prepping the quality on the right level

**Pull request merged**